rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -167,11 +167,17 @@ BoolFalse.  BoolLiteral ::= "false" ;
 -- Ground types:
 -- The "Literal" suffix avoids collisions with Simple Types
 GroundBool.    Ground ::= BoolLiteral ;
+GroundBigInt.  Ground ::= "BigInt" "(" BigIntLiteral ")" ;
 GroundInt.     Ground ::= LongLiteral ;
 GroundString.  Ground ::= StringLiteral ;
 GroundUri.     Ground ::= UriLiteral ;
 token StringLiteral ( '"' ((char - ["\"\\"]) | ('\\' ["\"\\nt"]))* '"' );
 token UriLiteral ('`' ((char - ["\\`"]) | ('\\' ["`\\"]))* '`') ;
+
+-- BigInt
+BigIntPositive. BigIntLiteral ::= LongLiteral ;
+BigIntNegative. BigIntLiteral ::= "-" LongLiteral ;
```

Comment:
We don't actually have `nagative Int` before. `-` is just a normal operator. Maybe we should add `Negative Int`, too.

Reply:
We should ask this to Greg in the next standup.

Reply:
We should ask this to Greg in the next standup.

Reply:
Yes, I think so too, I forgot about this question last standUp.

Reply:
I was in a meeting with Greg and he confirmed that we will not add variant with minus prefix. It's because minus is defined polymorphically on the process level so `-BigInt(42)` is a valid syntax.

Reply:
@tgrospic If we will create BigInt(-42) as -BigInt(42) we spend double times, and double account cost (because we need call negation()), can I save functional BigInt(-42) for BigInt type?